src/Settings/Default.hs

```diff
@@ -215,6 +215,52 @@ defaultBuilderArgs = mconcat
     , makeBuilderArgs
     , tarBuilderArgs ]
 
+-- | Disable some warnings in packages we use
+-- | https://github.com/ghc/ghc/blob/master/mk/warnings.mk#L46
+disableWarningArgsStage0 :: Args
+disableWarningArgsStage0 = stage Stage0 ? builder Ghc ? mconcat
+    [ package transformers ? append [ "-fno-warn-unused-matches", "-fno-warn-unused-imports" ]
+    , package terminfo ? append [ "-fno-warn-unused-imports" ] ]
+
+-- NOTE(izgzhen): The commented-out packages are NOT built by default with make either,
```

Comment:
I would prefer not to have them at all for now, because I'm not sure how up-to-date these settings are in Make. These packages are not built by default and the settings could be outdated.